src/js/angular/core/directives/rdfresourcesearch/templates/rdfResourceSearchTemplate.html

```diff
@@ -1,6 +1,6 @@
 <link href="css/rdf-resource-search.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
 
-<div ng-if="getActiveRepository() && !isLoadingLocation() && hasActiveLocation()" ng-cloak>
+<div ng-if="getActiveRepository() && hasActiveLocation() && isNotLoading()" ng-cloak>
```

Comment:
Why do you hide it on the repositories page?

Reply:
Exactly the opposite. Remains visible, even when triggered every n-th second events chain loads the repository location.